modules/govuk_rabbitmq/manifests/logging.pp

```diff
@@ -5,23 +5,48 @@
     fields  => {'application' => 'rabbitmq'},
   }
 
+  @filebeat::prospector { 'rabbitmq_startup_log':
+    paths  => '/var/log/rabbitmq/startup_log',
+    fields => {'application' => 'rabbitmq'},
+  }
+
   govuk_logging::logstream { 'rabbitmq_startup_error':
     logfile => '/var/log/rabbitmq/startup_error',
     fields  => {'application' => 'rabbitmq'},
   }
 
+  @filebeat::prospector { 'rabbitmq_startup_error':
+    paths  => '/var/log/rabbitmq/startup_error',
```

Comment:
similarly I think these could all be combined into a single `filebeat::prospector` by passing all the filenames to a single `paths` list